### PR TITLE
fix: WAFV2 Managed Rules PK

### LINKED ISSUE #847: WAFV2 Managed Rules PKs
### Describe the bug

When grabbing all Managed Rules from all regions and scopes there is a PK collision

The PK does not include scope so the results from the following API Calls will be the same:

`aws wafv2 list-available-managed-rule-groups --scope CLOUDFRONT --region us-east-1`
`aws wafv2 list-available-managed-rule-groups --scope REGIONAL --region us-east-1`

### Expected Behavior

We should be able to store the data cleanly in the db

### Steps to Reproduce

cloudquery fetch

### Possible Solution

_No response_

### Provider and CloudQuery version

all?

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by bbernays**
🎉 Thank you for making CloudQuery awesome by submitting a PR 🎉

#### Summary

---

Use the following steps to ensure your PR is ready to be reviewed

- [x] Read the [contribution guidelines](../CONTRIBUTING.md) 🧑‍🎓
- [x] Run `go fmt` to format your code 🖊
- [x] Update or add tests. Learn more about testing [here](https://docs.cloudquery.io/docs/developers/sdk/testing) 🧪
- [x] Update the docs by running `go run ./docs/docs.go` and committing the changes 📃
- [x] If adding a new resource, add [relevant Terraform files](../terraform) in a separate PR 📂
- [x] Ensure the status checks below are successful ✅

### CODE REVIEW

File: resources/provider/migrations/timescale/29_v0.11.3.up.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+-- Autogenerated by migration tool on 2022-05-04 19:52:11
+
+-- Resource: wafv2.managed_rule_groups
```
Comment (roneli) [inline resources/provider/migrations/timescale/29_v0.11.3.up.sql:3]: missing truncate

**Review by roneli — CHANGES_REQUESTED** [review]
Looks good, 1 comment.

**Review by roneli — APPROVED** [review]
LGTM, maybe would also truncate on downs to be safe?

**Review by disq — COMMENTED** [review]
web_acl has child tables, why not `TRUNCATE CASCADE`?

**Comment by bbernays** [comment]
> web_acl has child tables, why not `TRUNCATE CASCADE`?

Didn't realize that was an option. Have removed the `DELETE` and replaced with `TRUNCATE CASCADE`

**Pull request merged**